lib/ClusterShell/NodeSet.py

```diff
@@ -269,7 +269,7 @@ def _iter_nd_pat(self, pat, rset):
                 # set of user-provided fold axis (support negative numbers)
                 fold_axis = [int(x) % dimcnt for x in self.fold_axis
                              if -dimcnt <= int(x) < dimcnt]
-        except (TypeError, ValueError), exc:
+        except (TypeError, ValueError) as exc:
```

Comment:
Hey @wt thanks for this, appreciated! Now I know this can be frustrating, but `except ... as` only works from Python 2.6 and CS 1.7 still supports Python 2.4 - the current syntax works from Python 2.4 to 2.7. We plan to drop Python 2.4 (so also to drop EPEL 5 support) in ClusterShell 1.8+ and work on Python 3 support. It may be time to create a branch, as we have still a few bugfix patches to land in a v1.7.2...